presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -678,7 +688,7 @@ public void testArrayTimestampWithTimeZone(boolean insertWithPresto)
                 .addRoundTrip(dataType, asList(timeDoubledInJvmZone.atZone(UTC)));
         if (!insertWithPresto) {
             // Postgres results with non-DST time (winter time) for timeDoubledInJvmZone.atZone(jvmZone) while Presto results with DST time, so skip this test when insertWithPresto
-            // TODO: Possible fix:  columnValuesWithNames in io.prestosql.tests.datatype.CreateAsSelectDataSetup#literalInExplicitCast should contains timestamp values in utc. Same as with valueLiterals in io.prestosql.tests.datatype.CreateAndInsertDataSetup#insertRows
+            // Possible fix:  columnValuesWithNames in io.prestosql.tests.datatype.CreateAsSelectDataSetup#literalInExplicitCast should contains timestamp values in utc. Same as with valueLiterals in io.prestosql.tests.datatype.CreateAndInsertDataSetup#insertRows
```

Comment:
I meant removing the whole line (not just the word "TODO"). Sorry for not being clear.

Reply:
done